README.md

```diff
@@ -1,18 +1,22 @@
 # dlp-cli installation
 
-## Unix installation (Linux, MacOS)
+## Unix installation (Linux, MacOS, WSL)
 
-Step 1: Install homebrew (homebrew only works on mac and linux, so windows users may need WSL, may look into supporting winget in the future). You can find installation instructions for homebrew at [https://brew.sh/](https://brew.sh/)
+Step 0: If on Windows, install [WSL](https://learn.microsoft.com/en-us/windows/wsl/install)
+
+Step 1: Install homebrew. You can find installation instructions for homebrew at [https://brew.sh/](https://brew.sh/)
 
 Step 2: Run `brew tap DSGT-DLP/dlp-cli` in the terminal
 
 Step 3: Run `brew install dlp-cli` in the terminal
 
-## Windows installation
+## Windows installation (Do this if WSL doesn't work)
+
+Step 1: If you don't have the `dlp-cli` submodule, run `git submodule init dlp-cli` in the project directory. Then run `git submodule update`. If it says that you are in a detached HEAD state, cd to the dlp-cli submodule directory and `git checkout main`.
 
-Step 1: If you don't have the `dlp-cli` submodule, run `git submodule init dlp-cli` in the project directory. Then run `git submodule update`.
+Step 2: Run `winget settings --enable LocalManifestFiles` as administrator. Then, in the project directory, run `winget install -m dlp-cli/manifests/d/DSGT-DLP/dlp-cli/{version}`. Replace `{version}` with the version number that you want (typically the latest version).
 
-Step 2: In the project directory, run `winget install -m dlp-cli/manifests/d/DSGT-DLP/dlp-cli/{version}`. Replace `{version}` with the version number that you want (typically the latest version).
+Step 3: Add the path to the install location to your PATH environment variable (somewhere within `%localappdata%\microsoft\winget\packages`, may need file explorer to find the exact folder)
```

Comment:
i think the %localappdata thingy is part of the path variable. pelase make sure the casing is correct bc path variables are usually case sensitive @dwu359 @noah-iversen

Reply:
Noah said that it was correct

Reply:
nice